doc/flatpak-document-export.xml

```diff
@@ -91,30 +91,72 @@
 
                 <listitem><para>
                   Grant read access to the specified application.
+                  This option can be used multiple times.
+                </para></listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--allow-read</option></term>
+
+                <listitem><para>
+                  Grant read access to the applications specified with --app.
+                  This defaults to TRUE.
+                </para></listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--forbid-read</option></term>
+
+                <listitem><para>
+                  Revoke read access for the applications specified with --app.
                 </para></listitem>
             </varlistentry>
 
             <varlistentry>
                 <term><option>--allow-write</option></term>
 
                 <listitem><para>
-                  Also grant write access to the applications specified with --app.
+                  Grant write access to the applications specified with --app.
+                </para></listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--forbid-write</option></term>
+
+                <listitem><para>
+                  Revoke write access for the applications specified with --app.
                 </para></listitem>
             </varlistentry>
 
             <varlistentry>
                 <term><option>--allow-delete</option></term>
 
                 <listitem><para>
-                  Also grant the ability to delete a document id to the applications specified with --app.
+                  Grant the ability to delete a document id to the applications specified with --app.
```

Comment:
Maybe we should clarify that this doesn't mean deleting the file, but the whole document export.